bindings/c/test/unit/setup_tests.cpp

```diff
@@ -20,7 +20,7 @@
 
 // Unit tests for API setup, network initialization functions from the FDB C API.
 
-#define FDB_API_VERSION 720
+#define FDB_API_VERSION 730
```

Comment:
If now we are switching main version to 7.3, probably it would be better to make a separate version bump PR